core/src/main/resources/hudson/tasks/Fingerprinter/config.properties

```diff
@@ -1,17 +1,17 @@
 # The MIT License
-# 
-# Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
-# 
+#
+# Copyright (c) 2019, Mit Naria <mit4dev@gmail.com>
```

Comment:
Removing copy-pasted copyright from other files. Same goes for other license files.